hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ dependencies {
     api("org.hypertrace.core.grpcutils:grpc-context-utils:0.12.1")
     api("org.hypertrace.core.grpcutils:grpc-client-utils:0.12.1")
     api("org.hypertrace.core.grpcutils:grpc-client-rx-utils:0.12.1")
-    api("org.hypertrace.gateway.service:gateway-service-api:0.2.25")
+    api("org.hypertrace.gateway.service:gateway-service-api:0.3.0-SNAPSHOT")
```

Comment:
Once I merged the gateway-service PR and cut the release, will update the appropriate version here.